src/symbolize/gimli.rs

```diff
@@ -219,6 +221,7 @@ cfg_if::cfg_if! {
     } else if #[cfg(any(
         target_os = "macos",
         target_os = "ios",
+        target_os = "visionos",
         target_os = "tvos",
         target_os = "watchos",
     ))] {
```

Comment:
```suggestion
    } else if #[cfg(target_vendor = "apple")] {
```

Reply:
I made the change, but it seems that `target_vendor` will be deprecated at some point https://github.com/rust-lang/rust/issues/100343

Reply:
That initiative has made almost no motion forward in the past 2 years because no one has paused to address the real concerns involved.

So, effectively: I don't really care? On the day it does matter, it should be a deprecation notice first, and on that day I will update this code.